javascript/src/ycell.ts

```diff
@@ -187,7 +191,7 @@ export class YBaseCell<Metadata extends nbformat.IBaseCellMetadata>
   }
 
   /**
-   * Cell notebook awareness or null if the cell is standalone.
+   * Cell notebook awareness or null.
```

Comment:
Unrelated, but I spotted that if the cell is standalone then `this._awareness` is defined in line 184 (old line 180) so it definitely is not `null`.